src/_cffi_src/openssl/pem.py

```diff
@@ -44,6 +44,8 @@
 
 X509_CRL *PEM_read_bio_X509_CRL(BIO *, X509_CRL **, pem_password_cb *, void *);
 
+X509 *PEM_read_bio_X509_AUX(BIO *, X509 **, pem_password_cb *, void *);
```

Comment:
Can you drop this one. With https://github.com/python/cpython/pull/15303's merge, it should be possibel to use `PEM_read_bio_X509` in cryptography, which we already have.

Reply:
Fixed in pypy 7aed2cf1c9c0.

Reply:
I will update the PR once I get a successful pypy build